### PR TITLE
Operator can delete job runs

### PR DESCRIPTION
**Pull request by rupurt**
[Finishes #160398052]

### CODE REVIEW

File: gui/src/containers/Configuration/DeleteJobRuns.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        errored job runs. This action will delete runs a week ago
+        and before.
+      </Typography>
+
+      <Divider className={classes.deleteRunsDivider} />
+
+      <Grid container spacing={16}>
+        <Grid item xs={12}>
+          {!showCompletedConfirm &&
+            <Slide direction='right' in={!showCompletedConfirm} mountOnEnter unmountOnExit>
```
Comment (NavyAdmiral) [inline gui/src/containers/Configuration/DeleteJobRuns.js]: Why did we start to use animations? I don't remember seeing animations before here and seeing these two buttons slide in on Config page seems extra to me

File: gui/src/containers/Configuration/DeleteJobRuns.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+                Confirm delete all completed job runs up to {updatedBefore}
+              </Button>
+            </Slide>
+          }
+        </Grid>
+        <Grid item xs={12}>
+          {!showErroredConfirm &&
+            <Slide direction='right' in={!showErroredConfirm} mountOnEnter unmountOnExit>
+              <Button variant='outlined' onClick={() => setErroredConfirm(true)}>
+                Delete Errored Jobs
```
Comment (NavyAdmiral) [inline gui/src/containers/Configuration/DeleteJobRuns.js]: Normalize both buttons with same widths?

File: gui/src/containers/Configuration/DeleteJobRuns.js (hunk 1 above)
Comment (rupurt) [reply gui/src/containers/Configuration/DeleteJobRuns.js]: Our designer wants to use animations throughout the app. I chose to add this animation as it's a simple extra wrapper and the seed was planted when pairing on the style guide.

File: gui/src/containers/Configuration/DeleteJobRuns.js (hunk 2 above)
Comment (rupurt) [reply gui/src/containers/Configuration/DeleteJobRuns.js]: @NavyAdmiral pinged our designer.

Comment (NavyAdmiral) [reply gui/src/containers/Configuration/DeleteJobRuns.js]: @rupurt 
image
Delete buttons have the same width in the design

Comment (thoughtmerchant) [reply gui/src/containers/Configuration/DeleteJobRuns.js]: @rupurt @NavyAdmiral depending on the size of the viewport, the column will scale so we don't have total control of the width. For a stop-gap solution to delete jobs the button changing in size to confirm is not really a something we want to dump time into. 

Down the road, we chatted about a modal confirm with a more detailed description of what is going to happen with a clear confirm and cancel.

**Comment by j16r** [comment]
Merging on behalf of @rupurt since he's on vacay.

**Pull request merged**